query/tables/product_ingredient.py

```diff
@@ -0,0 +1,124 @@
+from asyncpg import Connection
+
+from query.database import get_rows_affected
+
+
+async def create_table(connection):
+    await connection.execute(
+        """create table product_ingredient (
```

Comment:
I don't remember why off-query deals with the ingredients structure?

Reply:
Neither can I. I think I was using it as a way to get different statistics for the recipe estimator